src/gurobi_logtools/api.py

```diff
@@ -28,7 +28,7 @@
 from gurobi_logtools.parsers.single_log import SingleLogParser
 
 
-class ParseResult:
+class FullLogParseResult:
```

Comment:
I don't love the name, but I can't think of a clearly better suggestion. Let's keep it.